### ISSUE TITLE
allowUntrustedAttestation = true doesn't seem to do anything 

### ISSUE THREAD

**Issue by AdamUCF** [Yubico/java-webauthn-server#16]
My rp instantiation is as follows...

```
        rp = RelyingParty.builder()
            .identity(rpId)
            .credentialRepository(this.userStorage)
            .origins(Sets.newHashSet(baseUrl))
            .attestationConveyancePreference(Optional.of(AttestationConveyancePreference.DIRECT))
            .metadataService(Optional.empty())
            .allowUnrequestedExtensions(true)
            .allowUntrustedAttestation(true)
            .validateSignatureCounter(true)
            .build();
```
When I attempt rp.FinishRegistration() I get the following exception which I wouldn't expect since it should allow untrusted attestation. I also get failures with TPM attestation (which I know isn't supported yet but would equally expect it to ignore as well).

com.yubico.webauthn.exception.RegistrationFailedException: java.lang.IllegalArgumentException: Failed to obtain attestation trust anchors.

**Comment by emlun** [comment]
Thanks for the report!

**Comment by emlun** [comment]
Should be fixed in version 1.0.1, publish to Maven Central in progress.

**Comment by AdamUCF** [comment]
Awesome! Thank you!
